libs/features/personalization/personalization.js

```diff
@@ -1037,15 +1047,22 @@ export const combineMepSources = async (persEnabled, promoEnabled, mepParam) =>
       }
     });
   }
-  return persManifests;
+  return initialPersManifestsPromises.concat(persManifests
+    .filter((m) => !('disabled' in m) || !m.disabled)
```

Comment:
if i remember correct, we shouldn't filter out disabled manifests.
those are 'promotions' manifests that are schedule is in the future/past, but Content Authors need to see them in the Page mep preview interface. However we don't load the .json itself (at least this is how it used to work) unless the manifest is active.

if you are still able to see a disabled manifest in the page then we are good